runtime/kernel/pcie/pcie_irq.h

```diff
@@ -139,6 +139,14 @@
 	_INTR(126)                                                             \
 	_INTR(127)
 
+#define TLKM_AWS_EC2_SLOT_INTERRUPTS                                           \
```

Comment:
Please run clang-format over your code, e.g. with
`find kernel -iname *.h -o -iname *.c | xargs clang-format -i`

The style files are checked in and should automatically be used.

Reply:
Even better if you could apply it to all your commits:

https://stackoverflow.com/questions/43974371/run-git-clang-format-on-series-of-git-commits

Reply:
I have opted for an additional commit to fix the formatting, because modifying the history at this point seems no longer feasible.